### PR TITLE
don´t assume service subnet is always the same

### PR DESCRIPTION
**Pull request by aojea**
The apiservers allocates a free ClusterIP for a Service if this field is not set.
To make the tests more portable, we can´t assume that we know the service-subnet.
Another option is to add the service subnet as a parameter and modify the IPs based on that

### CODE REVIEW

**Comment by aojea** [comment]
/assign @deads2k 
/hold
I don´t know how to regenerate the go bindata :shrug: , just opening the PR for discussion

File: test/extended/testdata/cmd/test/cmd/observe.sh [hunk 1]
```diff
@@ -28,8 +28,9 @@ os::cmd::expect_failure_and_text 'oc observe services --exit-after=1m --all-name
 os::cmd::expect_failure_and_text 'oc observe services --exit-after=1m --all-namespaces --retry-on-exit-code=2 --maximum-errors=1 --loglevel=4 -- /bin/sh -c "exit 2"' 'retrying command: exit status 2'
 
 # argument templates
-os::cmd::expect_success_and_text 'oc observe services --once --all-namespaces -a "{ .spec.clusterIP }"' '172.30.0.1'
-os::cmd::expect_success_and_text 'oc observe services --once --all-namespaces -a "{{ .spec.clusterIP }}" --output=gotemplate' '172.30.0.1'
+# The ClusterIP of the kubernetes.default service will always end in .1
```
Comment (soltysh) [inline test/extended/testdata/cmd/test/cmd/observe.sh:31]: If you want to update these tests it should be done in https://github.com/openshift/origin/blob/master/test/extended/cli/observe.go This repo is using outdated version which was a bash script which as you see is not anymore.

**Review by soltysh — CHANGES_REQUESTED** [review]

File: test/extended/testdata/cmd/test/cmd/observe.sh (hunk 1 above)
Comment (aojea) [reply test/extended/testdata/cmd/test/cmd/observe.sh:31]: oh, the full context is here http://post-office.corp.redhat.com/archives/aos-devel/2021-January/msg00248.html , then I guess the person that send the email should use the new tests, right?

Comment (soltysh) [reply test/extended/testdata/cmd/test/cmd/observe.sh:31]: I doubt this still holds, not sure what openshift-tests is atm, but I'm pretty confident that disconnected changes landed in origin.

Comment (aojea) [reply test/extended/testdata/cmd/test/cmd/observe.sh:31]: then is better to keep this as is and ask people to migrate, so it can die peacefully ;)

**Comment by aojea** [comment]
Thanks

**Pull request closed without merging**